security/dsa_scc.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: security.openshift.io/v1
+allowHostDirVolumePlugin: false
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: false
+allowPrivilegedContainer: false
+allowedCapabilities: 
+- SYS_RAWIO
```

Comment:
Can we add the reason why we need SYS_RAWIO permission in the commit log please?

Reply:
Will update it